egs/wsj/s5/utils/copy_data_dir.sh

```diff
@@ -86,10 +86,16 @@ fi
 
 if [ -f $srcdir/segments ]; then
   utils/apply_map.pl -f 1 $destdir/utt_map <$srcdir/segments >$destdir/segments
-  cp $srcdir/wav.scp $destdir
+  if [ -f $srcdir/wav.scp ]; then
+    cp $srcdir/wav.scp $destdir
+  elif [ -f $srcdir/images.scp ]; then
+    cp $srcdir/images.scp $destdir
+  fi
```

Comment:
Since we don't use a segments file in OCR recipes, we might not even need to add these changes. i.e. when there is a segments file, then it's not an OCR data dir. @danpovey, what do you think?

Also, I suggested editing `utils/copy_data_dir.sh` instead of creating a new `image/copy_data_dir.sh` because I expected the changes to be small. @danpovey what would you suggest?